spec/lib/vanagon/project/dsl_spec.rb

```diff
@@ -25,6 +25,14 @@
       proj.version_from_git
       expect(proj._project.version).to eq('1.2.3.1234')
     end
+    it 'sets the version based on the git describe with multiple dashes' do
+      expect(Vanagon::Driver).to receive(:configdir).and_return(configdir)
+      proj = Vanagon::Project::DSL.new('test-fixture', {})
+      proj.instance_eval(project_block)
+      expect(Vanagon::Utilities).to receive(:git_version).with(File.expand_path('..', configdir)).and_return('1.2.3---1234')
+      proj.version_from_git
+      expect(proj._project.version).to eq('1.2.3.1234')
```

Comment:
This spec test is duplicated here so when merged it'll be in here twice. Do you want to fix that up, or should we merge and fix it up in the next commit?